src/components/calcite-inline-editable/readme.md

```diff
@@ -47,6 +47,13 @@ There is no need to set a theme or scale on the `<calcite-inline-editable>` comp
 | `loading`            | `loading`              | specify whether the confirm button should display a loading state, defaults to false                                                                                   | `boolean`             | `false`                    |
 | `scale`              | `scale`                | specify the scale of the inline-editable component, defaults to the scale of the wrapped calcite-input or the scale of the closest wrapping component with a set scale | `"l" \| "m" \| "s"`   | `undefined`                |
 
+## Events
+
+| Event                              | Description                                  | Type                |
+| ---------------------------------- | -------------------------------------------- | ------------------- |
+| `calciteInlineEditableEditCancel`  | Emitted when the cancel button gets clicked. | `CustomEvent<void>` |
+| `calciteInlineEditableEditConfirm` | Emitted when the check button gets clicked.  | `CustomEvent<void>` |
```

Comment:
@benelan What do you think of the wording here?

Reply:
Its a little funky, maybe "Emits after clicking the xyz button"?